### DISCUSSION TITLE
不支持玩客云

### DISCUSSION THREAD

**Discussion by sdlkhfksl** [jxxghp/MoviePilot#2636] [问题咨询]
![屏幕截图 2024-08-10 135050](https://github.com/user-attachments/assets/48825ccc-4b8c-4dc6-bb2c-bffd13dee0a7)

**Comment by Cp0204** [comment]
不支持就是不支持，不要拿些老掉牙的设备来为难开发者。

**Comment by chenmdg** [comment]
你找迅雷给你开发一个不就行了吗？
